reactive/jenkins.py

```diff
@@ -230,3 +248,34 @@ def update_nrpe_config(nagios):
 def stop():
     service_stop("jenkins")
     status_set("maintenance", "Jenkins stopped")
+
+
+@when("reset.jenkins.storage")
+def reset_jenkins_storage():
+    status_set("maintenance", "Configuring Jenkins Storage")
+
+    jenkins_installed = get_state("apt.installed.jenkins")
+    if jenkins_installed:
+        service_stop('jenkins')
```

Comment:
Can't we avoid this check by requiring this reactive state hook to be fired only once jenkins is installed?

For example

```python
@when("jenkins-storage-attached", "apt.installed.jenkins")
def attach():
    # ...
```